DOWNSTREAM_OWNERS_ALIASES

```diff
@@ -2,15 +2,12 @@
 
 aliases:
   kni-scheduler-reviewers:
-    - MarSik
     - ffromani
-    - yanirq
-    - Tal-or
-    - jlojosnegros
     - swatisehgal
+    - Tal-or
   kni-scheduler-approvers:
-    - MarSik
     - ffromani
-    - yanirq
-    - Tal-or
+    - MarSik
     - swatisehgal
+    - Tal-or
+    - yanirq
```

Comment:
approvers will get reviewers rights and assignments by inheritance right ?  aka no need to duplicate entires

Reply:
I guess I can simplify this indeed